Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Toy/speedloaders.yml

```diff
@@ -23,7 +23,7 @@
 
 - type: entity
   id: CapLoader
-  name: "capgun loader"
+  name: "cap gun loader"
```

Comment:
"speed loader (cap)" would me more consistent, same question as with the ammo boxes though.

Reply:
Yeah I was just making a minor change because when you search for cap gun this doesn't usually show up. I think this way is nice as it clearly can't be confused with real weapons e.g. "Speed loader xx"